tests/v2/validation/rbac/rbac.go

```diff
@@ -237,6 +238,19 @@ func createRole(client *rancher.Client, context string, roleName string, rules [
 
 }
 
+func CheckRBACResourceExistsForUser(client *rancher.Client, collection *v1.SteveCollection, userName string) (bool, error) {
+	userId, err := users.GetUserIDByName(client, userName)
+	if err != nil {
+		return false, err
+	}
+	for _, item := range collection.Data {
```

Comment:
May be we can just move this part to rbac_test.go and add this as a private function? thoughts?

Reply:
Agree. For now, we can keep this as a private function since this is relevant to the rbac test. Moved this function to rbac_test.go